spa/src/components/settings/Integration/Integration.styled.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+`;
+
+export const Content = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 40px;
+
+  & [data-testid^='integration-card'] {
+    width: 314px;
+  }
```

Comment:
```suggestion
  display: grid;
  gap: 40px;
  grid-template-columns: repeat(auto-fit, 314px);
```

This way you don't have to target a testid.